### PR TITLE
Add project to repo root

### PR DESCRIPTION
**Pull request by coquer**
## Description

Remove the need to have a `.zip` file into the root of the repository, making it easier to download the sample code, because when you click download in github it will create zip file with the application source code, this will make a zip file inception.

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	
+		<div id="header" class="page">
+			<h1>Get in touch</h1>
+		</div>
+	
+		<div id="content">
+			<p>Give it a try, we hear it's all the rage these days!</p>
+			
+			<div id="contact">
+				<form action="http://formspree.io/wondercodersdk@gmail.com" method="post">
```
Comment (nlshopova) [inline contact.html]: can you please change the action, otherwise the email will get spammed?

**Review by nlshopova — CHANGES_REQUESTED** [review]
Thanks for the PR btw :) Just one small change would be good.

File: contact.html (hunk 1 above)
Comment (coquer) [reply contact.html]: Yep I will, when I looked at the code didnt notice that part :dancer:

**Comment by coquer** [comment]
No need for this PR since @pcherneva fix the zip

**Pull request closed without merging**